nautilus_core/execution/src/messages/mod.rs

```diff
@@ -33,6 +33,7 @@ pub use self::{
 };
 
 #[derive(Clone, Debug, Display)]
+#[allow(clippy::large_enum_variant)] // TODO: why?
```

Comment:
Yes this is a problem that will have to be fixed throughout the system. We'll tackle this once significant control flow is in Rust.